src/components/button/CdrButton.vue

```diff
@@ -1,3 +1,79 @@
+<script setup>
```

Comment:
Nice. The `<script setup>` syntax is a lot easier to parse and remember what goes where. It's definitely an improvement over what we're replacing.